lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -256,15 +256,6 @@ def _set_sgi_ipv4_and_v6_ingress_flows(self):
             actions = "output:LOCAL"
             self._install_flow(flows.MEDIUM_PRIORITY + 1, match, actions)
 
-        # forward the node solicite msg to host and UE
```

Comment:
@pshelar could you confirm that this is fine

Reply:
right, we can not remove this.

Reply:
@pruthvihebbani  why test fails with the flow installed?

Reply:
![image](https://user-images.githubusercontent.com/38202471/174535132-71c9a0d2-3d95-4b9e-afc5-100bf9279d3e.png)
Test fails with snapshot mismatch error